### DISCUSSION TITLE
Translate WYSIWYG heading tooltips

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#12580] [Feature Requests]
How to set WYSIWYG icon's titles  in another language? I mean when the mouse hovers over the icons of the toolbar.
Setting the Options Override of the WYSIWYG Interface with
```
{
    "language": "xx_XX"
}
```
did not make it.

Finally it looks like this Options Override has no effect (even after downloading the xx_XX.js package from TinyMCE site).
In contrast, it is the user's language that affects the icon's titles.

The problem is that only 4 icon's titles are translated (links, image, media and code). The others stay in English.

**Comment by rijkvanzanten** [comment]
The titles injected by tinymce aren't translated by Directus' translations. We'll have to look into how to dynamically load the correct TinyMCE language pack based on the current user's language to get those translated 🤔 https://www.tiny.cloud/get-tiny/language-packages/ / https://www.tiny.cloud/docs-4x/configure/localization/

**Reply by rijkvanzanten** [reply]
Lets treat this as the feature request for that 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
